src/api/auth/jest.config.e2e.js

```diff
@@ -3,7 +3,7 @@ const baseConfig = require('../../../jest.config.base');
 module.exports = {
   ...baseConfig,
   rootDir: '../../..',
-  setupFiles: ['<rootDir>/jest.setup.js', '<rootDir>/src/api/auth/jest.setup.js'],
+  setupFiles: ['<rootDir>/jest.setup.js'],
```

Comment:
I think you can remove this, since the `baseConfig` has `setupFiles: ['<rootDir>/jest.setup.js'],` in it, and you're merging that in on line 4.